### PR TITLE
I added the search html file but couldn't get any of the gs code to work

### PR DESCRIPTION
**Pull request by biniona**

### CODE REVIEW

File: Search.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <base target="_top">
+    <script>   
+    function handleFormSubmit(formObject) {
+        Logger.log("HELLO WORLD");
```
Comment (miekof) [inline Search.html]: Remove this log or make it more meaningful.

File: Search.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        google.script.run.withSuccessHandler(display).processSearch(formObject);
+    }
+    </script>
+  </head>
+  
+  <body>
+    <form onsubmit="handleFormSubmit(this)">
+        ISBN: <input type="search" name="ISBN"> <br>
```
Comment (miekof) [inline Search.html]: Can you update the input fields based on the design spec?

File: Code.gs [hunk 3]
```diff
@@ -94,6 +110,18 @@ function getBookInfoWithIsbn(isbn){
 function parseMyList(list) {
 }
 
+//prase list items to comma separated string
```
Comment (miekof) [inline Code.gs:203]: this function is already in the master branch. Let's figure out how to rebase the code next Tuesday.

**Review by miekof — CHANGES_REQUESTED** [review]
Can you also include Issue ID's in the git log? 
I can sow you how next Tuesday.

**Comment by biniona** [comment]
Sounds good! I don't think I have much time this upcoming week to make the
requested changes, but I am looking forward to going back and reworking the
code next Tuesday or if i find time.

Best, Alek

**Pull request merged**